core/services/relay/evm/mercury/config_digest.go

```diff
@@ -61,10 +61,11 @@ func configDigest(
 		// assertion
 		panic("copy too little data")
 	}
-	if types.ConfigDigestPrefixEVM != 1 {
-		// assertion
-		panic("wrong ConfigDigestPrefix")
-	}
+	// TODO return this check once types.ConfigDigestPrefixMercury is exported from libocr
+	// if types.ConfigDigestPrefixMercury != 6 {
+	// 	// assertion
+	// 	panic("wrong ConfigDigestPrefix")
+	// }
```

Comment:
Please uncomment already, unless you're waiting for libocr to be merged before you merge this